### ISSUE TITLE
World Ping not Being Displayed Through World Hopper Pluggin

### ISSUE THREAD

**Issue by burningwrath** [runelite/runelite#15723]
### Type

Other

### Operating System

Windows

### OS Version

windows 11

### Bug description

I dont know hen this started happening, but the ping setting on my world hopper task bar does not display the current ping on the right hand side for any of the worlds. I've tried reseting the pluggin, changing the settings to display current ping, unistalling/reinstalling the runelight application, and logging in/our of my account. It apprears that there is no fix for this bug, and it would be very useful to have thing ping of the world noted through runelight.

### Logs

[client_2022-08-26.0.log](https://github.com/runelite/runelite/files/9639993/client_2022-08-26.0.log)
[client_2022-08-27.0.log](https://github.com/runelite/runelite/files/9639994/client_2022-08-27.0.log)
[client_2022-08-28.0.log](https://github.com/runelite/runelite/files/9639995/client_2022-08-28.0.log)
[client_2022-08-29.0.log](https://github.com/runelite/runelite/files/9639996/client_2022-08-29.0.log)
[client_2022-08-30.0.log](https://github.com/runelite/runelite/files/9639997/client_2022-08-30.0.log)
[client_2022-09-01.0.log](https://github.com/runelite/runelite/files/9639998/client_2022-09-01.0.log)
[client_2022-09-02.0.log](https://github.com/runelite/runelite/files/9639999/client_2022-09-02.0.log)
[client_2022-09-03.0.log](https://github.com/runelite/runelite/files/9640000/client_2022-09-03.0.log)
[client_2022-09-04.0.log](https://github.com/runelite/runelite/files/9640001/client_2022-09-04.0.log)
[client_2022-09-05.0.log](https://github.com/runelite/runelite/files/9640002/client_2022-09-05.0.log)
[client_2022-09-06.0.log](https://github.com/runelite/runelite/files/9640003/client_2022-09-06.0.log)
[client_2022-09-07.0.log](https://github.com/runelite/runelite/files/9640004/client_2022-09-07.0.log)
[client_2022-09-08.0.log](https://github.com/runelite/runelite/files/9640005/client_2022-09-08.0.log)
[client_2022-09-09.0.log](https://github.com/runelite/runelite/files/9640006/client_2022-09-09.0.log)
[client_2022-09-10.0.log](https://github.com/runelite/runelite/files/9640007/client_2022-09-10.0.log)
[client_2022-09-11.0.log](https://github.com/runelite/runelite/files/9640008/client_2022-09-11.0.log)
[client_2022-09-12.0.log](https://github.com/runelite/runelite/files/9640009/client_2022-09-12.0.log)
[client_2022-09-13.0.log](https://github.com/runelite/runelite/files/9640010/client_2022-09-13.0.log)
[client_2022-09-14.0.log](https://github.com/runelite/runelite/files/9640011/client_2022-09-14.0.log)
[client_2022-09-15.0.log](https://github.com/runelite/runelite/files/9640012/client_2022-09-15.0.log)
[client_2022-09-17.0.log](https://github.com/runelite/runelite/files/9640013/client_2022-09-17.0.log)
[client_2022-09-18.0.log](https://github.com/runelite/runelite/files/9640014/client_2022-09-18.0.log)
[client_2022-09-19.0.log](https://github.com/runelite/runelite/files/9640015/client_2022-09-19.0.log)
[client_2022-09-20.0.log](https://github.com/runelite/runelite/files/9640016/client_2022-09-20.0.log)
[client_2022-09-21.0.log](https://github.com/runelite/runelite/files/9640017/client_2022-09-21.0.log)
[client_2022-09-22.0.log](https://github.com/runelite/runelite/files/9640018/client_2022-09-22.0.log)
[client_2022-09-23.0.log](https://github.com/runelite/runelite/files/9640019/client_2022-09-23.0.log)
[launcher_2022-08-26.0.log](https://github.com/runelite/runelite/files/9640020/launcher_2022-08-26.0.log)
[launcher_2022-08-27.0.log](https://github.com/runelite/runelite/files/9640021/launcher_2022-08-27.0.log)
[launcher_2022-08-28.0.log](https://github.com/runelite/runelite/files/9640022/launcher_2022-08-28.0.log)
[launcher_2022-08-29.0.log](https://github.com/runelite/runelite/files/9640023/launcher_2022-08-29.0.log)
[launcher_2022-08-30.0.log](https://github.com/runelite/runelite/files/9640024/launcher_2022-08-30.0.log)
[launcher_2022-09-01.0.log](https://github.com/runelite/runelite/files/9640025/launcher_2022-09-01.0.log)
[launcher_2022-09-02.0.log](https://github.com/runelite/runelite/files/9640026/launcher_2022-09-02.0.log)
[launcher_2022-09-03.0.log](https://github.com/runelite/runelite/files/9640027/launcher_2022-09-03.0.log)
[launcher_2022-09-04.0.log](https://github.com/runelite/runelite/files/9640028/launcher_2022-09-04.0.log)
[launcher_2022-09-05.0.log](https://github.com/runelite/runelite/files/9640029/launcher_2022-09-05.0.log)
[launcher_2022-09-06.0.log](https://github.com/runelite/runelite/files/9640030/launcher_2022-09-06.0.log)
[launcher_2022-09-07.0.log](https://github.com/runelite/runelite/files/9640031/launcher_2022-09-07.0.log)
[launcher_2022-09-08.0.log](https://github.com/runelite/runelite/files/9640032/launcher_2022-09-08.0.log)
[launcher_2022-09-10.0.log](https://github.com/runelite/runelite/files/9640033/launcher_2022-09-10.0.log)
[launcher_2022-09-11.0.log](https://github.com/runelite/runelite/files/9640034/launcher_2022-09-11.0.log)
[launcher_2022-09-12.0.log](https://github.com/runelite/runelite/files/9640035/launcher_2022-09-12.0.log)
[launcher_2022-09-13.0.log](https://github.com/runelite/runelite/files/9640036/launcher_2022-09-13.0.log)
[launcher_2022-09-14.0.log](https://github.com/runelite/runelite/files/9640037/launcher_2022-09-14.0.log)
[launcher_2022-09-15.0.log](https://github.com/runelite/runelite/files/9640038/launcher_2022-09-15.0.log)
[launcher_2022-09-17.0.log](https://github.com/runelite/runelite/files/9640039/launcher_2022-09-17.0.log)
[launcher_2022-09-18.0.log](https://github.com/runelite/runelite/files/9640040/launcher_2022-09-18.0.log)
[launcher_2022-09-19.0.log](https://github.com/runelite/runelite/files/9640041/launcher_2022-09-19.0.log)
[launcher_2022-09-20.0.log](https://github.com/runelite/runelite/files/9640042/launcher_2022-09-20.0.log)
[launcher_2022-09-21.0.log](https://github.com/runelite/runelite/files/9640043/launcher_2022-09-21.0.log)
[launcher_2022-09-22.0.log](https://github.com/runelite/runelite/files/9640044/launcher_2022-09-22.0.log)
[launcher_2022-09-23.0.log](https://github.com/runelite/runelite/files/9640045/launcher_2022-09-23.0.log)

**Comment by geheur** [comment]
are you able to ping other things? e.g. run `ping 8.8.8.8` in cmd. If that doesn't work then something in your network is probably blocking ping/icmp.

**Comment by burningwrath** [comment]
Yeah I can run the cmnd for pinging my server and it has no issues.

**Comment by capSAR273** [comment]
This is happening regularly across multiple days still.

**Comment by Cabbage-Bomb** [comment]
completely broken for me as well. does not show on plugins tab, does not show on top right.

<img width="241" height="947" alt="Image" src="https://github.com/user-attachments/assets/0d01ad79-6910-45eb-9402-e5643a963149" />
